food-truck-api/src/main/java/food/truck/api/search/TruckSearchService.java

```diff
@@ -31,7 +31,7 @@
 
         Query truckQuery = qb.keyword()
                 .fuzzy().withEditDistanceUpTo(2)
-                .onFields("name", "description") // TODO: Search in tags, as well
+                .onFields("name", "description", "tags") 
```

Comment:
I'm guessing the annotations on the DB side of things were already in place, so we just needed to specify?